test-data/unit/check-kwargs.test

```diff
@@ -491,3 +491,28 @@ m = {} # type: Mapping[str, object]
 f(**m)
 g(**m)  # TODO: Should be an error
 [builtins fixtures/dict.pyi]
+
+
+[case testGivingArgumentAsPositionalAndKeywordArg3]
```

Comment:
Should also error if you pass a kwargs that doesn't exist (e.g., you pass a dict with keys `Literal["c", "d"]` to a function defined as `def f(a, b)`).

Reply:
Hey @JelleZijlstra, for that case, should the error message be `Unexpected keyword argument` or `Missing positional argument`?

Reply:
Probably both? It should match what we do if you just call that function as `f(c=..., d=...)`.